src/Sulu/Bundle/AdminBundle/Resources/public/js/vendor/husky/husky.js

```diff
@@ -29546,7 +29525,7 @@ define('husky_components/datagrid/decorators/table-view',[],function() {
          * @param record {Object} the data of the record
          */
         executeRowPostRenderActions: function(record) {
-            if (!!this.datagrid.itemIsSelected.call(this.datagrid, record.id)) {
+            if (record.selected === true) {
```

Comment:
This should not be rewritten... Is that just an old version or have you changed that?